core-services/prow/02_config/_config.yaml

```diff
@@ -11,19 +11,11 @@ deck:
     header_color: black
     logo: /static/extensions/logo.png
   default_rerun_auth_configs:
-  - repo: '*'
+  - repo: smg247/ci-docs
     rerun_auth_configs:
-      github_users:
-      - stevekuznetsov
-      - droslean
-      - bbguimaraes
-      - hongkailiu
-      - vikaslaad
-      - dobbymoodge
-      - alvaroaleman
-      - bparees
-      - AlexNPavel
-      - jupierce
+      github_team_slugs:
+      - org: smg247-org
+        slug: sample
   - repo: openshift/openshift-tests-private
```

Comment:
what're these changes?

Reply:
it looks like it's just been moved from an autogeneration 

https://github.com/openshift/release/pull/28107/files/97d6a7e9fdb7a71e9fef93a7d38295222838fcef#diff-3fe4522c3fa00ecd918a986bc31b4cde479c02bcace2b0497bd136760edbd71fL37